app/assets/stylesheets/components/_youtube-player.scss

```diff
@@ -45,10 +45,15 @@
   &:hover {
     color: govuk-colour('white');
   }
+  &:hover:focus,
+  &:hover:focus .govuk-details__summary-text {
+    color: govuk-colour('black');
+  }
 }
 
 .govuk-details__text {
   color: govuk-colour('white');
+  border-left-color: govuk-colour('white')
```

Comment:
Could we add a semicolon at the end of this line please (sorry!)

Reply:
Sorry! It's been a while since I made any changes to scss files :-)